sklearn/metrics/tests/test_pairwise_distances_reduction.py

```diff
@@ -216,101 +219,92 @@ def assert_radius_neighbors_results_quasi_equality(
     """
     is_sorted = lambda a: np.all(a[:-1] <= a[1:])
 
-    n_significant_digits = -(int(floor(log10(abs(rtol)))) + 1)
-
     assert (
-        len(ref_dist) == len(dist) == len(ref_indices) == len(indices)
+        len(neighbors_dists_a)
+        == len(neighbors_dists_b)
+        == len(neighbors_indices_a)
+        == len(neighbors_indices_b)
     ), "Arrays of results have various lengths."
```

Comment:
```suggestion
    )
```

This way we are sure that the pytest informative message is not shadowed by our custom non informative message.